cookbook/console/logging.rst

```diff
@@ -81,7 +81,7 @@ First configure a listener for console exception events in the service container
 
         # app/config/services.yml
         services:
-            kernel.listener.command_dispatch:
```

Comment:
Unrelated to this pull request, but shouldn't these services be called `app.*` instead of `kernel.*` ?